src/neo/Persistence/ISnapshot.cs

```diff
@@ -8,7 +8,7 @@ namespace Neo.Persistence
     public interface ISnapshot : IDisposable, IReadOnlyStore
     {
         void Commit();
-        void Delete(byte table, byte[] key);
-        void Put(byte table, byte[] key, byte[] value);
+        void Delete(byte[] key);
```

Comment:
How should the missing `table` field be handled in LevelDB plugin? I.e. when key is null, what should func `CreateKey` return?

Reply:
Then `key` should not be `null`.

Reply:
If we use `StorageKey` and `StorageValue` it will be more eficient for MPT